src/style.css

```
@@ -1 +1,15 @@
-/* styles go here */
+label {
+  display: block;
+}
+
+fieldset {
+  margin: 20px;
+}
+
+input {
+  margin: 10px 0;
+}
+
+.inline > * {
+  display: inline;
+}
```

```suggestion
.inline > * {
  display: inline;
}
```

Do you really need these styles?